sql/synth_recipes.sql

```diff
@@ -4752,6 +4752,8 @@ INSERT INTO `synth_recipes` VALUES (75528,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
 INSERT INTO `synth_recipes` VALUES (75529,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4157,0,0,2779,2779,2779,2779,1,1,1,1,'Enfeeb. Kit Poison');
 INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4163,0,0,2780,2780,2780,2780,1,1,1,1,'Enfeeb. Kit Blind');
 INSERT INTO `synth_recipes` VALUES (75531,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4161,0,0,2781,2781,2781,2781,1,1,1,1,'Enfeeb. Kit Sleep');
+INSERT INTO `synth_recipes` VALUES (75532,1,0,0,0,24,0,0,0,0,0,4098,4240,1632,0,0,0,0,0,0,0,661,661,661,752,2,2,2,1,'Moblin Mail (desynth)');
+INSERT INTO `synth_recipes` VALUES (75533,1,0,0,0,23,0,0,0,0,0,4098,4240,1625,0,0,0,0,0,0,0,661,661,661,752,1,1,1,1,'Moblin Helm (desynth)');
```

Comment:
These should go on the bottom of their respective range of IDs for the craft. Each craft is broken out into "ID Brackets" 1-10k, 10-20k, etc.